test/gitea_test.go

```diff
@@ -210,7 +210,7 @@ func TestGiteaConcurrencyExclusivenessMultipleRuns(t *testing.T) {
 
 func TestGiteaRetestAfterPush(t *testing.T) {
 	topts := &tgitea.TestOpts{
-		Regexp:      regexp.MustCompile(`.*pr has.*failed`),
+		Regexp:      regexp.MustCompile(`.*has <b>failed</b>`),
```

Comment:
How it use to pass earlier :thinking: 
Is there something changes happened in GITEA ?

Reply:
we changed the output of the comment, before it will show the pipelinerunname as defined on the .tekton/ yaml file (originalprname) now we are showing the actual pipelinerun (with the random bits at the end).... 

there is another test fix later on, (the exit 1) which i don't know how come it wasn't failing before.... it's perhaps as well new version of pipeline that shows proper error now..

Reply:
Ah i see :+1: